### PR TITLE
feat: adding grafana tutorial

### PR DESCRIPTION
**Pull request by deimosfr**

### CODE REVIEW

File: website/guides/tutorial/grafana-install.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+     website/guides/tutorial/grafana-install.md.erb
+-->
+
+## Grafana setup
+
+First of all, create a project and an environment. Then let's create Grafana application.
+
+<Alert type="info">
+
+At the moment, Qovery does not support configuration file injection into Docker. So it can't be connected to an external database.
```
Comment (alecarrano) [inline website/guides/tutorial/grafana-install.md:32]: *So the Grafana application can't be connected to an external database via a dedicated configuration file

File: website/guides/tutorial/grafana-install.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+<p Valign="center">
+  <img src="/img/grafana/qovery_create_app.png" alt="create gafana app" />
+</p>
+
+1. Set an application name
+2. Select the application source type: Container registry
+3. Select DockerHub Public (or the one you have. If you do not have one, create a registry pointing to DockerHub)
+4. Set the image name: grafana/grafana
+5. Take a look at the latest [Grafana tags](https://hub.docker.com/r/grafana/grafana/tags) and set the tag you want to use
```
Comment (alecarrano) [inline website/guides/tutorial/grafana-install.md:49]: note: do not use "latest"

**Review by alecarrano — COMMENTED** [review]
small updates

File: website/guides/tutorial/grafana-install.md (hunk 1 above)
Comment (deimosfr) [reply website/guides/tutorial/grafana-install.md:32]: Not easily until we support env files

Comment (alecarrano) [reply website/guides/tutorial/grafana-install.md:32]: it was a suggestion to update the text 🤦

**Pull request merged**